### DISCUSSION TITLE
When will data table be available?

### DISCUSSION THREAD

**Discussion by JenuelDev** [unovue/reka-ui#553] [Q&A] [answered]
When will data table be available?

**Comment by sadeghbarati** [comment, marked as answer]
Check out [`shadcn-vue`](https://www.shadcn-vue.com/docs/components/data-table.html) Data Table

Which is using [`@tanstack/vue-table`](https://tanstack.com/table/v8/docs/adapters/vue-table)

**Reply by zernonia** [reply]
We dont have plan for data table yet. And as @sadeghbarati  suggested, you can check out the [@tanstack/vue-table](https://tanstack.com/table/v8/docs/adapters/vue-table) 😁

**Comment by kusman28** [comment]
Do we need to install shadcn to use the tanstack/vue-table?

**Reply by ReturnMeVoid** [reply]
no
